protostar/cheatable_starknet/cheaters/contracts.py

```diff
@@ -253,14 +218,28 @@ async def call(
         contract_address: Address,
         function_name: str,
         cheaters: "CairoCheaters",
+        unregister_expected_call: Callable,
         calldata: Optional[CairoOrPythonData] = None,
     ) -> CairoData:
         contract_address_int = int(contract_address)
-        cairo_calldata = await self._transform_calldata_to_cairo_data_by_addr(
-            contract_address=contract_address,
+        cairo_calldata = await transform_calldata_to_cairo_data(
+            contract_class=await self.cheatable_state.get_contract_class(
+                class_hash=await self.cheatable_state.get_class_hash_at(
+                    contract_address=contract_address_int
+                )
```

Comment:
this is repeated too many times, I'd definitely create a method like `get_contract_class_by_address`

Reply:
[add get_contract_class_by_address](https://github.com/software-mansion/protostar/pull/1363/commits/856826bf8d1207436a370d8aa5ea9dae60670a62)